### DISCUSSION TITLE
Moving Predbat docker container to another server

### DISCUSSION THREAD

**Discussion by jasonwragg** [springfall2008/batpred#1535] [General]
Hi,

Maybe a silly question, I am running predbat in a docker container using docker compose. I want to move it to another server, do I just copy the contents of my config folder to the new server? 

Thanks

**Comment by nipar4** [comment]
Depending on how the image was built, you just need your apps.yaml

**Reply by jasonwragg** [reply]
So it doesnt store any data anywhere else like a database somewhere?

**Reply by nipar4** [reply]
No that I'm aware of, there was talk of a database but I haven't heard anything since.
I rebuild and move predbat all the time and haven't noticed any lose of data
If you want to be sure just copy the config directory over, that won't hurt either

**Reply by gcoan** [reply]
there is work started on a predbat database but its early days.  At the moment Predbat doesn't really retain anything from one run to another, and such that it does, it uses Home Assistant sensors to record history
